src/components/scenes/LoginScene.js

```diff
@@ -44,6 +45,10 @@ class LoginSceneComponent extends Component<Props, State> {
     slowlog(this, /.*/, global.slowlogOptions)
   }
 
+  componentDidMount () {
+    this.props.loggedOut()
```

Comment:
I could be wrong, but it seems like this should only fire one time, when the app launches. What happens if you log in & log out a bunch of times? Does the login scene get re-mounted each time?

Reply:
Verified that it does, in fact, re-fire.